### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by ninetyninedt** [pypi/support#5372]
### PyPI Username

[ninetyninedt]

### Reason for request

When I try to log in, the system says there’s no account with that username, but I know I’m using the username I used when I created the account. 

### Recovery Codes

- [x] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [x] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Thespi-Brain** [comment]
Hi @ninetyninedt,

I did not find an account with the username you provided for the main PyPI account, but I did, however, find an account with that username for test.pypi. If you think you definitely have a PyPI account besides the test one, please send the email address associated with that account to support@pypi.org and reference this link (https://github.com/pypi/support/issues/5372). We will proceed from there. Thanks!

**Comment by ninetyninedt** [comment]
Yep, that’s what the problem was. I created the account on test.pypi, and not on the main site. Boy, do I feel stupid. Thank you very much for sorting me out and I’m sorry for the trouble I caused. On Jan 2, 2025, at 1:13 PM, thespibrain ***@***.***> wrote:﻿
Hi @ninetyninedt,
I did not find an account with the username you provided for the main PyPI account, but I did, however, find an account with that username for test.pypi. If you think you definitely have a PyPI account besides the test one, please send the email address associated with that account to ***@***.*** and reference this link (#5372). We will proceed from there. Thanks!

**Comment by Thespi-Brain** [comment]
No worries, I am glad we got it sorted out. Closing the issue!
